todoapp/app/build.gradle

```diff
@@ -12,11 +12,20 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
+
+        jackOptions {
+            enabled true
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
     }
 
     buildTypes {
         debug {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
Yes. With `minifyEnabled true`, when building, you get the error 
`Minifying the variant used for tests is not supported when using Jack.`